components/navbar/navbar.snip.html

```diff
@@ -57,9 +58,14 @@
     {{/sidebaronly}}
     {{#fixeditems}}
     <div class="ampstart-headerbar-fixed center m0 p0 mr2 flex justify-center nowrap">
-      {{#calltracking}}
-        {{> ../calltracking/calltracking.snip.html}}
-      {{/calltracking}}
+      <div class="ml1">
```

Comment:
I think it should be mr2 - (you can drop the mr2 on the ampstart-headerbar-fixed)